client/app/dbaas/metrics/metrics.constant.js

```diff
@@ -0,0 +1,20 @@
+angular.module("managerApp")
+    .constant("METRICS_ENDPOINTS", {
+        graphs: [
+            {
+                name: "Quantum",
+                url: "https://quantum.metrics.ovh.net"
+            },
+            {
+                name: "Grafana",
+                url: "https://grafana.tsaas.ovh.com"
+            }
+        ],
+        protos: [
+        // 'prometheus',
```

Comment:
I won't touch that, until documentation is fully completed and clearly defined by metrics team.